app/pages/home-for-user/news-pullout.jsx

```diff
@@ -101,6 +101,10 @@ const NewsSection = React.createClass({
   render() {
     const projLink = `${window.location.origin}/projects/${this.state.newestProject.slug}`;
     const avatarSrc = !!this.state.newestAvatar ? this.state.newestAvatar.src : null;
+    const recentProjects = 
+      this.props.updatedProjects
+      .sort((a, b) => { return new Date(b.updated_at) - new Date(a.updated_at); })
```

Comment:
Is this necessary? When the initial API call is made to retrieve project preferences, it is sorted by `-updated_at`. I assume that would do the sorting for you, but I could be missing something here.